layouts/about-release-schedule.hbs

```diff
@@ -14,7 +14,7 @@
         {{{ contents }}}
 
         <p>
-          <img alt="{{ title }}" src="https://raw.githubusercontent.com/nodejs/Release/master/schedule.png">
+          <img alt="{{ title }}" src="https://raw.githubusercontent.com/nodejs/Release/master/schedule.svg?sanitize=true">
```

Comment:
What is the `?sanitize=true` doing? 🤔

Reply:
Maybe refs: https://stackoverflow.com/a/47952195/8102333

Reply:
Hi,
Because if I don't use that parameter, the response type from the server would be 'text/plain', and if I use that, it will return me the right content-type.

Compare them together :)

【Before using parameter】
![wrong](https://user-images.githubusercontent.com/40081831/52520863-91e04500-2ca9-11e9-9e99-6d9c8e816d1e.PNG)
【After using parameter】
![right](https://user-images.githubusercontent.com/40081831/52520870-af151380-2ca9-11e9-841b-6b01f91c45a5.PNG)

Reply:
I'm not sure whether this is the problem related to the server-side, it seems that the server doesn't return me the right response of content-type until I add this parameter.

Reply:
Or @fhemberger, any other better ideas? I've already tested and compared XD

Reply:
I've updated my submit commit in details to explain this :)

Reply:
Awesome, didn't knew that trick. Let's do this!